<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoAIFacialRig</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="head_name">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="head_x">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="head_y">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="head_z">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="set_head_button">
      <property name="text">
       <string>Set Head</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="create_head_bb_button">
      <property name="text">
       <string>Create Head Bounding Box</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="create_camera_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Create Camera</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clean_up_viewport_button">
      <property name="text">
       <string>Clean Up Viewport</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="playblast_button">
      <property name="text">
       <string>Playblast</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ai_button">
      <property name="text">
       <string>AI</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>225</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
